CHANGELOG.md

```diff
@@ -3,15 +3,20 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 Use the following format for additions: ` - VERSION: [feature/patch (if applicable)] Short description of change. Links to relevant issues/PRs.`
 
+- 1.1.12:
+    - Refactor and promisify clicktests [#879](https://github.com/FredrikNoren/ungit/pull/879)
 - 1.1.11:
     - Fix cli arguments [#871](https://github.com/FredrikNoren/ungit/pull/871)
     - Stop if ~/.ungitrc contains syntax error
     - Removed official support ini format of ~/.ungitrc, because internal API supports only JSON
-- 1.1.10: Fix broken diff out in some cases when diff contains table. [#881](https://github.com/FredrikNoren/ungit/pull/881)
+- 1.1.10:
+    - Fix broken diff out in some cases when diff contains table. [#881](https://github.com/FredrikNoren/ungit/pull/881)
+    - Refactor to remove async and using more promises. [#878](https://github.com/FredrikNoren/ungit/pull/878)
```

Comment:
There were some releases since I have created this PR and made that changelog update, I will have to fix this.